packages/edit-site/src/components/global-styles/font-library-modal/style.scss

```diff
@@ -186,3 +186,15 @@ button.font-library-modal__upload-area {
 		justify-content: center;
 	}
 }
+
+.font-library-modal__select-all {
+	width: 100%;
+	height: auto;
+	padding: $grid-unit-20;
+
+	.components-checkbox-control__label {
+		padding-left: $grid-unit-20;
+		font-size: 18px;
+		line-height: 1;
```

Comment:
In my opinion, it seems fine to leave the font size and line height at the default values ​​of the `CheckboxControl` itself.

Reply:
That would appear strange because default size is too small in comparison to the list of font faces title.

Reply:
<img width="1013" alt="image" src="https://github.com/user-attachments/assets/f13fdee3-03ab-47c3-9bcc-1d347aa5c62e">

Reply:
Personally I don't mind, but what do you think, @jasmussen?

Reply:
I also think this is fine. We should avoid ad hoc font styles where possible.

Reply:
Font styles are removed now.